src/dotnet/commands/dotnet-compile-fsc/Program.cs

```diff
@@ -106,150 +104,184 @@ public static int Run(string[] args)
+                {
+                    var win32manifestPath = Path.Combine(AppContext.BaseDirectory, "default.win32manifest");
+                    allArgs.Add($"--win32manifest:{win32manifestPath}");
+                }
             }
 
-            // Additional arguments are added verbatim
-            if (options.AdditionalArguments != null)
+            if (commonOptions.SuppressWarnings != null)
             {
-                commonArgs.AddRange(options.AdditionalArguments);
+                allArgs.Add("--nowarn:" + string.Join(",", commonOptions.SuppressWarnings.ToArray()));
             }
 
-            if (options.LanguageVersion != null)
+            if (commonOptions.LanguageVersion != null)
             {
+                // Not used in fsc
             }
 
-            if (options.Platform != null)
+            if (commonOptions.Platform != null)
             {
-                commonArgs.Add($"--platform:{options.Platform}");
+                allArgs.Add($"--platform:{commonOptions.Platform}");
             }
 
-            if (options.AllowUnsafe == true)
+            if (commonOptions.AllowUnsafe == true)
             {
             }
 
-            if (options.WarningsAsErrors == true)
+            if (commonOptions.WarningsAsErrors == true)
             {
-                commonArgs.Add("--warnaserror");
+                allArgs.Add("--warnaserror");
             }
 
-            if (options.Optimize == true)
+            //set target framework
+            if (targetNetCore)
             {
-                commonArgs.Add("--optimize");
+                allArgs.Add("--targetprofile:netcore");
             }
 
-            if (options.KeyFile != null)
+            if (commonOptions.DelaySign == true)
             {
+                allArgs.Add("--delaysign+");
             }
 
-            if (options.DelaySign == true)
+            if (commonOptions.PublicSign == true)
             {
             }
 
-            if (options.PublicSign == true)
+            if (commonOptions.AdditionalArguments != null)
             {
+                // Additional arguments are added verbatim
+                allArgs.AddRange(commonOptions.AdditionalArguments);
             }
 
-            if (options.GenerateXmlDocumentation == true)
+            // Generate assembly info
+            var assemblyInfo = Path.Combine(tempOutDir, $"dotnet-compile.assemblyinfo.fs");
+            File.WriteAllText(assemblyInfo, AssemblyInfoFileGenerator.GenerateFSharp(assemblyInfoOptions));
+
+            //source files + assemblyInfo
+            allArgs.AddRange(GetSourceFiles(sources, assemblyInfo).ToArray());
+
+            //TODO check the switch enabled in fsproj in RELEASE and DEBUG configuration 
+
+            var rsp = Path.Combine(tempOutDir, "dotnet-compile-fsc.rsp");
+            File.WriteAllLines(rsp, allArgs, Encoding.UTF8);
+
+            // Execute FSC!
+            var result = RunFsc(new List<string> { $"@{rsp}" })
```

Comment:
response file is a new feature of fsc, was not supported with the old version.
but with command line are really nice for development/debug now